packages/runtime/test/src/interpreter/opcode-list.ts

```diff
@@ -217,11 +217,14 @@ describe("Teal Opcodes", function () {
 
   describe("Arg[N]", function () {
     const stack = new Stack<StackElem>();
-    const runtime = new Runtime([]);
-    const interpreter = new Interpreter();
+    let interpreter: Interpreter;
     const args = ["Arg0", "Arg1", "Arg2", "Arg3"].map(stringToBytes);
-    interpreter.runtime = runtime;
-    interpreter.runtime.ctx.args = args;
+
+    this.beforeAll(() => {
+      interpreter = new Interpreter();
+      interpreter.runtime = new Runtime([]);
+      interpreter.runtime.ctx.args = args;
+    });
```

Comment:
why do we need to recreate interpreter?

Reply:
I don't think we are. We are creating in `beforeAll` hook. Did you mean something else?

Reply:
right.